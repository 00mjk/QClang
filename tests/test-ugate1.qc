int main() {
    qubit q;
		q = false;
    U(6.2831, 3.14159, 1.5707, q);
    return 0;
}
