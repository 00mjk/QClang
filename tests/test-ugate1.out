OPENQASM 2.0;
include "qelib1.inc";
qreg q[1];
creg c[1];
h q;
qreg temp_0[1];
U(6.2831, 3.14159, 1.5707) temp_0;
